src/scripts.js

```diff
@@ -143,15 +151,51 @@ const displayByTag = (tag) => {
 }
 
 const createRecipeCardEventListener = () => {
-  const recipeCards = document.querySelectorAll('article');
+  createWholeCardListener();
+  createRecipeHeartListener();
+  createRecipePlusListener();
+}
+
+const createWholeCardListener = () => {
+  const recipeCards = document.querySelectorAll('.recipe-image');
   recipeCards.forEach(recipeCard => {
     recipeCard.addEventListener('click', (event) => {
-      console.log(event.target);
       displayBigModal(event);
     });
   });
 }
 
+const createRecipeHeartListener = () => {
```

Comment:
Not sure if it's good practice, but I love the method of querySelectorAll and iterating through to addEventListener to each of them!

One of my favorite pieces of the project thus far